changelog/Microsoft.DirectoryServices.json

```
@@ -10508,6 +10508,56 @@
       "CreatedDateTime": "2021-11-15T20:42:47.116895Z",
       "WorkloadArea": "Identity and access",
       "SubArea": "Identity and sign-in"
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "438838d1-a936-4789-9d1c-c929788be1eb",
+          "ApiChange": "EnumType",
+          "ChangedApiName": "federatedIdpMfaBehavior",
+          "ChangeType": "Addition",
+          "Description": "Added the **federatedIdpMfaBehavior** enumeration type.",
+          "Target": "federatedIdpMfaBehavior"
+        },
+        {
+          "Id": "438838d1-a936-4789-9d1c-c929788be1eb",
+          "ApiChange": "EnumType",
```

A changelog gen bug generated the incorrect ApiChange value. The fix is underway.
```suggestion
          "ApiChange": "Enum type",
          "ChangedApiName": "federatedIdpMfaBehavior",
          "ChangeType": "Addition",
          "Description": "Added the **federatedIdpMfaBehavior** enumeration type.",
          "Target": "federatedIdpMfaBehavior"
        },
        {
          "Id": "438838d1-a936-4789-9d1c-c929788be1eb",
          "ApiChange": "Enum type",
```